src/lab10.py

```diff
@@ -13,13 +13,14 @@
 from lab2 import WIDTH, HEIGHT, HSTEP, VSTEP, SCROLL_STEP
 from lab3 import FONTS, get_font
 from lab4 import print_tree, HTMLParser
-from lab5 import BLOCK_ELEMENTS, DrawRect, DocumentLayout
+from lab5 import BLOCK_ELEMENTS, DrawRect
 from lab6 import CSSParser, TagSelector, DescendantSelector
 from lab6 import INHERITED_PROPERTIES, style, cascade_priority
 from lab6 import DrawText, tree_to_list
-from lab7 import DrawLine, DrawOutline, LineLayout, TextLayout
+from lab7 import DrawLine, DrawOutline
 from lab8 import URL, Browser, Text, Element
 from lab8 import BlockLayout, InputLayout, INPUT_WIDTH_PX
+from lab8 import DocumentLayout, LineLayout, TextLayout
```

Comment:
Why remove these? Is this something in your editor that removes unused imports?

Reply:
I removed them because they are not used. Is this another case where you think something or other will be broken - the outline maybe?

Reply:
Yeah. I promise I will fix this to not be necessary once we ship the book text!

Reply:
Is it easy to thrown an error if this doesn't work in the inliner? How do we know we haven't missed other cases?

Reply:
oh now I remember. I removed some from lab5 because lab8 now overrides them. I
did not in fact remove anyy imports.